sphinx/themes/basic/static/basic.css_t

```diff
@@ -311,6 +311,16 @@ img.align-default, figure.align-default, .figure.align-default {
     text-align: right;
 }
 
+#skip {
```

Comment:
This should be skip-target, no?
But anyway I think you are trying to style the `a` link here though, not the actual element with the id?
Where did you get the css from? Most sources look to suggest adding more like position absolute: https://www.w3schools.com/accessibility/accessibility_skip_links.php

Reply:
Or also https://www.a11y-collective.com/blog/skip-to-main-content/ etc

Reply:
Oh yes, this should have `position: absolute`. I just threw this draft PR together to show that the solution doesn't require much code.

The selector should be `#skip`, not `#skip-target`. We can change the names to make their purposes more clear but `#skip` is the anchor with the text "skip to main content" that should be the first element after `<body>` to ensure that it always receives focus first. By default this element should be focusable but offscreen, that's why we push it `1000%` out of the visible viewport.

Reply:
I'm OK with adding the ID to an already existing element in the templates if that's what you all prefer. My original thinking is that if you create a new, empty target, there's a little less risk that someone will accidentally remove/change that element as opposed to the actual main content element.

Reply:
added `absolute` positioning and fixed the IDs